editors/intellij/gradle.properties

```diff
@@ -0,0 +1,34 @@
+# IntelliJ Platform Artifacts Repositories -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
+
+pluginGroup = com.github.biomejs.intellijbiome
+pluginName = Biome
+pluginRepositoryUrl = https://github.com/biomejs/biome
```

Comment:
Is there a documentation page to learn about these properties? I went in https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html and couldn't find anything about `pluginName` for example